Sources/main.swift

```diff
@@ -12,7 +12,7 @@ import SwiftGenKit
 
 // MARK: Common
 
-let templatesRelativePath = (Bundle.main.resourcePath ?? ".") + "/templates"
+let templatesRelativePath = "../templates"
```

Comment:
I'm already making a fix that uses flags added by the rake build command, we can then simply `#if` from swift code.